include/stl2/detail/range/access.hpp

```diff
@@ -415,49 +449,42 @@ STL2_OPEN_NAMESPACE {
 		template <class R>
 		constexpr bool has_member = false;
 		template <class R>
-		requires
-			requires(const R& r) {
-				{ r.empty() } -> bool;
+		requires requires(const R& r) {
+				// { r.empty() } -> bool;
+				requires ConvertibleTo<__f<decltype(r.empty())>, bool>;
```

Comment:
This is not to spec (and neither was `{ r.empty() } -> bool;`). This should be a simple expression requirement `bool(r.empty())`.